robotd/devices.py

```diff
@@ -97,6 +97,30 @@ def status(self):
         return {'temperature': temp_milli_degrees / 1000}
 
 
+class GameState(Board):
+    """
+    State storage for the game. keeps a store of whet it is sent
+    """
+
+    # define the name od the board
+    board_type_id = 'game'
+    create_on_startup = True
+
+    def __init__(self):
+        super().__init__({})
+        self._state = {'corner': 0, 'mode': 'development'}
```

Comment:
1. done
2. We'll need to keep the state of competition at some point if we want to have a 3 minute shut-off timer.
3. The 'zone' comment is fair, I will fix